lib/plugins/create/templates/aws-nodejs/serverless.yml

```diff
@@ -1,96 +1,74 @@
-# Welcome to Serverless!
-#
-# This file is the main config file for your service.
-# It's very minimal at this point and uses default values.
-# You can always add more config options for more control.
-# We've included some commented out config examples here.
-# Just uncomment any of them to get that config option.
-#
-# For full config options, check the docs:
-#    docs.serverless.com
-#
-# Happy Coding!
+## The serverless.yml file is the main config file for your service.
+## To see all available serverless.yml properties see the docs http://bit.ly/2rJJjMV
+## ───── Serverless docs https://serverless.com/framework/docs/ ─────
 
-service: aws-nodejs # NOTE: update this with your service name
-
-# You can pin your service to only deploy with a specific Serverless version
-# Check out our docs for more details
-# frameworkVersion: "=X.X.X"
+##───────────────────────────────────────────────────────────────────────
+## 'service' - The name of the service and addition meta information
+##───────────────────────────────────────────────────────────────────────
+service:
+  name: aws-nodejs
 
+##───────────────────────────────────────────────────────────────────────
+## 'provider' - Defines where your service will be deployed. ('aws', 'azure', 'etc')
+##───────────────────────────────────────────────────────────────────────
 provider:
   name: aws
   runtime: nodejs6.10
+##● Define additional provider properties to overwrite the defaults
+#   stage: dev
+#   region: us-east-1
+##● Service wide environment variables can be defined here
+#   environment:
+#     variable1: value1
+##● Custom IAM statements can be added to the functions in this service here
```

Comment:
IMHO the easiest, most minimalistic statement is smth. like https://github.com/serverless/examples/blob/808009c57daaa802feff32a9506010b4da8d317e/aws-python-rest-api-with-dynamodb/serverless.yml#L10-L19